hooks/990-ensure-uids-gids.chroot

```diff
@@ -20,20 +20,16 @@ echo "Ensure gshadow file does not change"
 diff -u /etc/gshadow /etc/gshadow.orig
 rc=$?; [ "$rc" != "0" ] && MISMATCH=1
 
-echo "Ensure extrausers passwd file does not change"
-diff -u /var/lib/extrausers/passwd /var/lib/extrausers/passwd.orig
+echo "Ensure no docker"
+! grep -q docker /etc/passwd /etc/shadown /etc/group /etc/gshadow
```

Comment:
mmm travis seems unhappy about this line

Reply:
works now.